packages/liveblocks-core/src/room.ts

```diff
@@ -881,7 +848,7 @@ function makeStateMachine<
       storageOperations: [],
     },
 
-    connection: new ValueRef<Connection>({ status: "closed" }),
+    connection: new ValueRef({ status: "closed" }),
```

Comment:
I know this isn't introduced in this PR, but should we use an enum/const here?

Reply:
Would probably be a good idea indeed. I may tackle this in an upcoming PR 👍